crates/server/src/auth/oidc.rs

```diff
@@ -199,26 +199,25 @@ impl Claims {
         scope_context: ScopeContext,
```

Comment:
Since this is a follow-up, let's also fix the misplaced/inconsistent `use` statements added here in https://github.com/profianinc/drawbridge/commit/bee4e85dd72eb27c91403b49dbddeb1f21d0fa3a# and https://github.com/profianinc/drawbridge/pull/306/files#r1023851691